lib/modules/syntaxHighlighting.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import hljs from 'highlight.js';
```

Comment:
Since this is a fairly large library which is requires a little processing when loaded, we might want to lazy load it. Take a look at how the dash.js library is loaded:
https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/5a36f5ab6dfbb48b661bc61b6b7a540ea17d9e78/lib/modules/showImages.js#L4
https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/5a36f5ab6dfbb48b661bc61b6b7a540ea17d9e78/lib/modules/showImages.js#L1972

Reply:
I was originally going to lazy load Highlight.js, but unfortunately no dist files are provided in the NPM package.